src/cdk/a11y/interactivity-checker/interactivity-checker.ts

```diff
@@ -9,6 +9,15 @@
 import {Platform} from '@angular/cdk/platform';
 import {Injectable} from '@angular/core';
 
+/**
+ * Configuration for the isFocusable method.
+ */
+export class IsFocusableConfig {
+  /**
+   * Whether to count an element as focusable even if it is not currently visible.
+   */
+  includeInvisible: boolean = false;
```

Comment:
I'd personally vote for `ignoreVisibility`, because `include` seems not really clear (personally to me). What do you think?

Reply:
I agree that ignoreVisibility is clearer. Changed!